apps/frontend/src/app/components/header/header.component.html

```diff
@@ -1,19 +1,47 @@
-<ix-application-header name="Xcelerator Demo App">
-	<ix-breadcrumb visibleItemCount="3">
+<ix-application class="h-screen">
+	<ix-application-header name="Xcelerator Demo App">
+		<button class="placeholder-logo" slot="logo" [routerLink]="['/']">
+			<img
+				src="https://cdn.c2comms.cloud/images/logo-collection/2.1/sie-logo-white-rgb.svg"
+				style="height: 20px"
```

Comment:
Bitte mach das styling entweder in scss zb durch classe oder nutze eine utility klasse von tailwind.

Reply:
hat Patrick gemacht